python/src/spark_rapids_ml/classification.py

```diff
@@ -975,6 +989,7 @@ def _single_fit(init_parameters: Dict[str, Any]) -> Dict[str, Any]:
                     "n_cols": logistic_regression.n_cols,
                     "dtype": logistic_regression.dtype.name,
                     "num_iters": logistic_regression.solver_model.num_iters,
+                    "objective": logistic_regression.solver_model.objective,
```

Comment:
Nice.   We can improve summary output with this.  I guess it is just final value not full history.

Reply:
Right, cuml currently only returns the final value. Need to update C++ side if to match the full summary